lib/locales/en.ts

```diff
@@ -972,6 +1008,7 @@ export const locale: Locale = {
     thomas_apostle: 'Saint Thomas, Apostle',
     thomas_aquinas_priest: 'Saint Thomas Aquinas, Priest and Doctor of the Church',
     thomas_becket_bishop: 'Saint Thomas Becket, Bishop and Martyr',
+    thomas_helye_priest: 'Blessed Thomas Helye, Priest',
```

Comment:
I personally would keep the acute accent.

```suggestion
    thomas_helye_priest: 'Blessed Thomas Hélye, Priest',
```

Reply:
Indeed.